app-config.yaml

```diff
@@ -4,6 +4,7 @@ app:
   branding:
     fullLogo: ${BASE64_EMBEDDED_FULL_LOGO}
     iconLogo: ${BASE64_EMBEDDED_ICON_LOGO}
+    fullLogoSize: ${FULL_LOGO_SIZE}
```

Comment:
will this be needed only for fullLogo? or icon logo as well

and in naming should we be more explicit i.e `fullLogoWidth`

Reply:
++ to `fullLogoWidth`

@invincibleJai I think the fixed width for `iconLogo` is good enough since it places the icon in the center of the collapsed sidebar.

The issue only occurs for `fullLogo`. But we can probably add support for iconLogo width for complete support.

Reply:
I have changed `fullLogoSize` to `fullLogoWidth`